### PR TITLE
ON-33: Delete instance

### PR DESCRIPTION
**Pull request by pankogito**
implementation of delete function instance

### CODE REVIEW

File: ontology_api/instance/instance_service.py [hunk 1]
```diff
@@ -53,3 +53,24 @@ def get_instance(self, model_id: int, class_name: str, instance_label: str):
...
+        owl_class = onto[class_name]
+        if owl_class is None:
+            onto.destroy()
+            return InstanceModelOut(errors="Class named " + str(class_name) + " not found in Model " + str(model_id))
+        for i in owl_class.instances():
+            if len(i.label) > 0 and i.label[0] == instance_label:
+                instance_id = i.name
+                destroy_entity(i)
+                onto.destroy()
+                return FullInstanceModelOut(instance_id=instance_id, label=instance_label)
```
Comment (KKucht) [inline ontology_api/instance/instance_service.py:73]: I don't see update ontology

File: ontology_api/instance/instance_service.py [hunk 2]
```diff
@@ -53,3 +53,24 @@ def get_instance(self, model_id: int, class_name: str, instance_label: str):
...
+        onto.destroy()
+        return InstanceModelOut(
+            errors="Instance with label " + str(instance_label) + " not found in Model " + str(model_id))
```
Comment (KKucht) [inline ontology_api/instance/instance_service.py:76]: It better looks, if the correct model is in the end.

**Review by KKucht — CHANGES_REQUESTED** [review]
Change pull name to ON-33: Delete instance

**Review by KKucht — APPROVED** [review]

**Review by mrwrob — CHANGES_REQUESTED** [review]

File: ontology_api/instance/instance_service.py [hunk 3]
```diff
@@ -53,3 +53,28 @@ def get_instance(self, model_id: int, class_name: str, instance_label: str):
         onto.destroy()
         return InstanceModelOut(
             errors="Instance with label " + str(instance_label) + " not found in Model " + str(model_id))
+
+    def delete_instance(self,model_id: int, class_name: str, instance_label: str):
```
[the remark this answers is not in the document]
Comment (PLangowski) [reply ontology_api/instance/instance_service.py]: @mrwrob Fixed.

**Review by mrwrob — APPROVED** [review]
Good job :+1:

**Pull request merged**